engine/apps/slack/models/slack_usergroup.py

```diff
@@ -112,11 +111,7 @@ def update_oncall_members(self):
     def update_members(self, slack_ids):
         sc = SlackClientWithErrorHandling(self.slack_team_identity.bot_access_token)
 
-        sc.api_call(
-            "usergroups.users.update",
-            usergroup=self.slack_id,
-            users=slack_ids,
-        )
```

Comment:
in `api_call()` the `timeout` parameter has been removed and is now passed at the client level ([migration guide docs](https://github.com/slackapi/python-slack-sdk/wiki/Migrating-to-2.x#web-client-api-changes))